ResidentsSocialCarePlatformApi.Tests/V1/E2ETests/GetCaseNote.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using ResidentsSocialCarePlatformApi.V1.Boundary.Responses;
+
+namespace ResidentsSocialCarePlatformApi.Tests.V1.E2ETests
+{
+    [TestFixture]
+    public class GetCaseNote : EndToEndTests<Startup>
+    {
+        [Test]
```

Comment:
Should we have a test to check that when no CaseNote is found we return a 404?

Reply:
Yes!

Reply:
@JohnFarrellDev Added an E2E test to cover this in https://github.com/LBHackney-IT/residents-social-care-platform-api/pull/11/commits/a46627e2b24e6b8b8ba68e785d692bc8ca539e7d.

Reply:
Perfect